### PR TITLE
Cookie warning

### LINKED ISSUE #21: Create cookie notification
**Task:**
Create cookie notification

**Steps:**
1. Using Bootstrap create popup message about cookies
2. Add one button “Allow”, close on click
3. Add translation and compile it

### PR DESCRIPTION
**Pull request by Owen-Dechow**
closes #21 
I added a cookie warning using bootstrap to the best of my ability. I added one new CSS rule to the main.css file and used Vanilla JS in the base.html file. I'm not familiar with Russian or Tartar so I used Google Translate (Google Translate sometimes miss translates in my experience, this doesn't happen often but it's worth mentioning).

Another thing worth mentioning is that after running the `makemigrations` command a new migration was created. I didn't edit any models so I'm assuming this was just an edit that was never migrated.

New CSS
```css
.cookie-allow-tag{
    bottom: 1em;
    max-width: 500px;
    width: 90%;
    z-index: 1;
}
```

New HTML
```html
    <div id="cookie-allow-tag" class="position-fixed cookie-allow-tag bg-dark flex-column p-3 d-flex flex-column rounded start-50 translate-middle-x">
        <span class="text-white text-center">{% translate 'CookieWarning' %}</span>
        <button class="btn btn-light mt-1" onclick='localStorage.setItem("cookie-allow-tag", "true"); this.parentNode.remove();'>{% translate 'CookieWarningAccept' %}</button>
    </div>

    <script>
        if (localStorage.getItem("cookie-allow-tag"))
        {
            document.getElementById("cookie-allow-tag").remove();
        }
    </script>
```

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -38,6 +38,19 @@
         {% block content %}{% endblock %}
     </main>
     {% include 'components/footer.html' %}
+
+    <div id="cookie-allow-tag" class="position-fixed cookie-allow-tag bg-dark flex-column p-3 d-flex flex-column rounded start-50 translate-middle-x">
```
Comment (roaddust2) [inline templates/base.html:42]: Duplicate class

File: locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -112,6 +112,14 @@ msgstr "Images"
 msgid "ErrorEmptyEmail"
 msgstr "Email field must not be empty"
 
+#: templates/base.html:43
+msgid "CookieWarning"
+msgstr "Warning! This website uses cookies. By using this site you are accepting all cookies."
```
Comment (roaddust2) [inline locale/en/LC_MESSAGES/django.po:117]: Our website uses cookies, by using it, you are accepting all cookies.

File: locale/en/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -112,6 +112,14 @@ msgstr "Images"
...
+
+#: templates/base.html:44
+msgid "CookieWarningAccept"
+msgstr "Accept Cookies"
```
Comment (roaddust2) [inline locale/en/LC_MESSAGES/django.po:121]: Accept

File: locale/ru/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -114,6 +114,14 @@ msgstr "Изображения"
 msgid "ErrorEmptyEmail"
 msgstr "Поле Email должно быть заполнено"
 
+#: templates/base.html:43
+msgid "CookieWarning"
+msgstr "Предупреждение! Этот веб-сайт использует файлы cookie. Используя этот сайт, вы принимаете все файлы cookie."
```
Comment (roaddust2) [inline locale/ru/LC_MESSAGES/django.po:119]: Наш веб-сайт использует файлы cookie, оставаясь на нем, вы принимаете все файлы cookie.

File: locale/ru/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -114,6 +114,14 @@ msgstr "Изображения"
...
+
+#: templates/base.html:44
+msgid "CookieWarningAccept"
+msgstr "Принимать файлы cookie"
```
Comment (roaddust2) [inline locale/ru/LC_MESSAGES/django.po:123]: Принять

File: locale/tt/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -112,6 +112,14 @@ msgstr "Сурәтләр"
 msgid "ErrorEmptyEmail"
 msgstr "Email язу кыры тутырылган булырга тиеш"
 
+#: templates/base.html:43
+msgid "CookieWarning"
+msgstr "Кисәтү! Бу сайт cookie'ларны куллана. Бу сайтны кулланып сез барлык кукиларны кабул итәсез."
```
Comment (roaddust2) [inline locale/tt/LC_MESSAGES/django.po:117]: Безнең вебсайт cookie куллана, безнең сайтта калып сез барлык cookie кабул итәсез.

File: locale/tt/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -112,6 +112,14 @@ msgstr "Сурәтләр"
...
+
+#: templates/base.html:44
+msgid "CookieWarningAccept"
+msgstr "Cookie'ларны кабул итегез"
```
Comment (roaddust2) [inline locale/tt/LC_MESSAGES/django.po:121]: Яхшы

**Comment by roaddust2** [comment]
Hello there, really appreciate your PR.
I added a few comments for translation strings, everything else looks fine. You also reminded me how important to check on `makemigrations` . If you make that changes, I'll be happy to merge PR.

**Comment by Owen-Dechow** [comment]
Thanks for your feedback, I expected some translation issues. I won't be able to make any fixes till the end of the week but, I'll get them fixed as soon as I can.

**Pull request merged**